apps/api_web/lib/api_web/api_controller_helpers.ex

```diff
@@ -171,7 +171,7 @@ defmodule ApiWeb.ApiControllerHelpers do
   end
 
   defp view_module_for_type(type) do
-    view_name = String.capitalize(type) <> "View"
+    view_name = Macro.camelize(type) <> "View"
```

Comment:
It doesn't look like this works: http://localhost:4000/live-facilities/park-alfcl-garage?fields[live-facility]=updated_at

Reply:
Oh, that looks like like something we should fix:
- https://api-v3.mbta.com/live-facilities/park-woodl-garage has type `live-facility`
- https://api-v3.mbta.com/route-patterns/Red-1-1 has type `route_pattern`

Can we standardize on `hyphen-separated` like the URLs?